### PR TITLE
Remove "reply" button for forum posts that have parents

### LINKED ISSUE #2003: Remove "reply" button for forum posts that have parents.
If a forum post a parent, it should not have a reply button there. This is to prevent a lot nesting of posts that are unrelated to the parent forum post.

### PR DESCRIPTION
**Pull request by nicholette**
screenshot 2017-02-08 14 38 20
Fixes #2003.

### CODE REVIEW

**Comment by allenwq** [comment]
@nicholette Don't need to open new PR, just rebase and force push. Otherwise, the review context gets lost. 

BTW, do you want to add a comment that @fonglh mentioned in the previous PR ?

**Comment by nicholette** [comment]
@allenwq I made a new pr because i renamed the branch name. The previous PR is #2008. 

There's not much in the comment by @fonglh except pointing out that old comments will still have their multiple indentation (e.g. 2 indents).

**Comment by allenwq** [comment]
He has one more comment in the outdated diff, you might missed that :-)

**Comment by nicholette** [comment]
Sorry, really missed the comment by @fonglh.

screen shot 2017-02-13 at 3 36 40 pm

**Comment by allenwq** [comment]
@nicholette the build failed, I think you should use single slash `/` for comments.

File: app/views/course/forum/posts/_post.html.slim [hunk 1]
```diff
@@ -19,9 +19,11 @@
       - if defined?(show_buttons)
         div.pull-right
           .btn-group
-            = link_to reply_course_forum_topic_post_path(current_course, @forum, @topic, post),
-                      title: t('.reply'), class: ['btn', 'btn-primary'] do
-              = fa_icon 'mail-reply'.freeze
+            // remove reply button to replies of post
```
Comment (fonglh) [inline app/views/course/forum/posts/_post.html.slim]: Comment character for slim is a single slash.

**Pull request merged**